recipes/tslearn/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tslearn" %}
+{% set version = "0.1.10.8" %}
+{% set sha256 = "7b1e22f93c107884fb104d5d25d37eb8772a341aeab91b3a22ce6d860ef4cde8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy 1.11.*
+  run:
+    - python
+    - numpy >=1.11
+    - scipy
+    - scikit-learn
```

Comment:
Surprising that it is not more sensitive to scikit-learn version. Guess if there are issues, upstream will hear about them as well.

Reply:
True. Well, also scikit-learn <0.17.1 is not currently available on conda forge, and 0.17.1 might be fine.

Reply:
>  also scikit-learn <0.17.1 is not currently available on conda forge

Since it's possible to mix different conda channels, never mind my comment.